docs/content/mongocxx-v3/installation/_index.md

```diff
@@ -36,6 +36,64 @@ The Mongo C++ driver is available in the following package managers.
 - [Vcpkg](https://vcpkg.io/) (search for mongo-cxx-driver)
 - [Conan](https://conan.io/center/recipes/mongo-cxx-driver)
 
+### Vcpkg Install Instructions
+
+If you do not already have Vcpkg installed, install it with the following
+command:
+```
+$ git clone https://github.com/Microsoft/vcpkg.git
+$ cd vcpkg
+$ ./bootstrap-vcpkg.sh
+```
+
+Optionally, to install with Visual Studio integration:
+```
+vcpkg integrate install
+```
+
+Install the driver.
+```
+$ ./vcpkg install mongo-cxx-driver
+```
+
+You can find the header files in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/include/
+```
+
+The library files are in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/lib/
+```
+
+### Conan Install Instructions
```

Comment:
Include an up-front notice hat the package specifier is `mongo-cxx-driver/<version>`. More advanced Conan users will look for this information specifically. Same recommendation for the vcpkg section.

Reply:
Got it. I added the following line for the Conan documentation, but since Vcpkg handles versioning entirely through git, what is your recommendation for Vcpkg?
```
Package Specifier: `mongo-cxx-driver/3.8.0`
```

Reply:
For vcpkg, just the package name is enough for anyone to get started quickly. Maybe leave a note that the version may be out of date on older vcpkg versions.